packages/suite/src/views/wallet/transactions/components/AccountEmpty.tsx

```diff
@@ -51,15 +51,15 @@ export const AccountEmpty = ({ account }: AccountEmptyProps) => {
                     key: '1',
                     onClick: handleNavigateToReceivePage,
                     children: (
-                        <Translation id="TR_RECEIVE_NETWORK" values={{ network: networkSymbol }} />
+                        <Translation id="TR_RECEIVE_NETWORK" values={{ network: symbolTicker }} />
                     ),
                 },
                 {
                     'data-testid': '@accounts/empty-account/buy',
                     key: '2',
                     onClick: handleNavigateToBuyPage,
                     children: (
-                        <Translation id="TR_BUY_NETWORK" values={{ network: networkSymbol }} />
+                        <Translation id="TR_BUY_NETWORK" values={{ network: symbolTicker }} />
```

Comment:
can we just do symbol: symbol.toUpperCase()?

Reply:
ok